gadget/gadget.go

```diff
@@ -704,9 +561,9 @@ func validateVolumeStructure(vs *VolumeStructure, vol *Volume) error {
 		return err
 	}
 
-	if err := validateReservedLabels(vs); err != nil {
+	/*if err := validateReservedLabels(vs); err != nil {
```

Comment:
This should be removed, I think.

Reply:
yes, sorry, it's removed in the follow-up